### ISSUE TITLE
AWS S3 Support

### ISSUE THREAD

**Issue by kidspro88** [x0b/rcx#85]
Do you have any plan to support AWS s3 in the next time?

**Comment by x0b** [comment]
In general, you can use any remote type, including Amazon S3, when you import a configuration file from a desktop rclone. 

The app does not (yet) have a configuration wizard for Amazon S3 though. It seems we haven't had the time yet, and also no test accounts.

**Comment by SkypLabs** [comment]
Some cloud providers other than AWS provide S3-compatible object storage services, such as [Scaleway](https://www.scaleway.com/en/object-storage/) and [OVH](https://www.ovhcloud.com/en/public-cloud/object-storage/).

The first 75GB are free on Scaleway, which would be way enough for running CI tests on it. I can help you set up a test project / account if you are interested.

**Comment by simson** [comment]
It works pretty well using the rclone configuration import.

**Comment by SkypLabs** [comment]
> It works pretty well using the rclone configuration import.

Yes it does, but adding an S3 remote directly from the app with a GUI would be handier though.

**Comment by dipankardas011** [comment]
Yes we need the GUI for this please, as most people use aws s3
